scripts/upload_pact_VT.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 set -e
 
+export PROJECT_VERSION="1.3.0"
```

Comment:
- Do we need to increase the version?
- What about renaming this into `VERSION` or `CONTRACT_VERSION` ?

Reply:
I have upgraded the version from 1.2.2 > 1.3.0 as we've added discovery test to the contract